internal/reader/ccip.go

```diff
@@ -53,6 +53,13 @@ type CCIP interface {
 		seqNumRange cciptypes.SeqNumRange,
 	) ([]cciptypes.Message, error)
 
+	// GetExpectedNextSequenceNumber returns the next sequence number to be used
+	// in the onramp.
+	GetExpectedNextSequenceNumber(
+		ctx context.Context,
+		sourceChainSelector cciptypes.ChainSelector,
```

Comment:
I thought we were going to stop using the implicit destination:

```suggestion
		source, dest cciptypes.ChainSelector,
```

Reply:
Good catch, done.